src/Tool/Npm.php

```diff
@@ -52,7 +52,26 @@ public static function build(
             'built project',
             'failed to build project'
         ))->run();
-        static::start($output, 'building project');
+    }
+
+    public static function run(OutputInterface $output, string $script): bool
+    {
+        $prefix = N::getPrefix($output);
+
+        // Note: script is not escaped so that args can be passed.
+        $command = match (true) {
+            (static::isPnpm()) => 'pnpm run ' . $script,
```

Comment:
I don't think the static calls, `static::isPnpm()`, `static::isYarn()` in the match's single conditional expression need to be wrapped in parentheses here.

Reply:
(had chat, needs to stay for now until prettier is implemented).